dns/dnscrypt-proxy/src/opnsense/service/templates/OPNsense/Dnscryptproxy/dnscrypt-proxy.toml

```diff
@@ -89,7 +89,7 @@ tls_disable_session_tickets = true
 tls_disable_session_tickets = false
 {%   endif %}
 
-fallback_resolver = '{{ OPNsense.dnscryptproxy.general.fallback_resolver }}'
+bootstrap_resolvers = '{{ OPNsense.dnscryptproxy.general.bootstrap_resolvers }}'
```

Comment:
probably best to only switch out the faulty config key

Reply:
As you wish :)